frontend/src/bundles/common/components/select/libs/components/dropdown-indicator.tsx

```diff
@@ -0,0 +1,13 @@
+import { components } from 'react-select';
+
+import arrowDown from '~/assets/img/icons/arrow-down-icon.svg';
+
+const DropdownIndicator: typeof components.DropdownIndicator = (properties) => {
```

Comment:
can you please replace it with : JSX.Element since this is the pattern we follow in other components